jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -349,9 +369,14 @@ private String debugId() {
     }
 
     @Override
-    public io.opentracing.Span start() {
+    public io.opentracing.Span startManual() {
       SpanContext context;
 
+      // Check if active span should be established as CHILD_OF relationship
+      if (!ignoreActiveSpan && null != activeSpanSource.activeSpan()) {
+        asChildOf(activeSpanSource.activeSpan());
+      }
```

Comment:
lgtm. Just one more thing - reading the [API docs](https://github.com/opentracing/opentracing-java/blob/master/opentracing-api/src/main/java/io/opentracing/Tracer.java#L172), the active span should only be used as a parent if there are no other references added explicitly. So need to add `&& references. isEmpty`